src/main/java/org/reaktivity/nukleus/tls/internal/TlsNukleus.java

```diff
@@ -138,48 +140,45 @@ private boolean handleRoute(
         return true;
     }
 
-    private void handleRoute(
+    private boolean handleRoute(
         final RouteFW route)
     {
         final TlsRouteExFW routeEx = route.extension().get(tlsRouteExRO::wrap);
         final String store = routeEx.store().asString();
         final long routeId = route.correlationId();
 
-        if (store != null)
+        storesByRouteId.put(routeId, store);
+        StoreInfo storeInfo = contextsByStore.computeIfAbsent(store, s -> initContext(config, store, ++storeIndex));
+        if (storeInfo != null)
         {
-            storesByRouteId.put(routeId, store);
+            storeInfo.routeCount++;
         }
-        MutableInteger routesCount = routesByStore.computeIfAbsent(store, s -> new MutableInteger());
-        routesCount.value++;
-        contextsByStore.computeIfAbsent(store, s -> initContext(config, store));
+
+        return storeInfo != null;
```

Comment:
Under what conditions is `storeInfo` `null` given the `computeIfAbsent` logic above?

Reply:
newStoreInfo() would return null, if there are more than 56 ca certs